modules/rostests/rosautotest/CWineTest.cpp

```diff
@@ -8,6 +8,7 @@
 #include "precomp.h"
 
 static const DWORD ListTimeout = 10000;
+static WCHAR TestName[] = L"rosautotest.exe";
```

Comment:
What if I happen to have copied and renamed this file to some other name, but otherwise try the setup you are trying to fix?

Reply:
See latest commit and let me know what you think. Thank.